packages/client/src/generation/TSClient/Input.ts

```diff
@@ -56,6 +57,10 @@ function stringifyInputType(
     return 'null'
   }
 
+  if (typeof type === 'string' && (type.endsWith('Select') || type?.endsWith('Include'))) {
```

Comment:
Question: if there possibility of collision here? Will we be safe if one of the model names ends with `Input`?

Reply:
Also, do `Select` types pass through `Input` class in general? From `Model.ts` it seems like they don't

Reply:
> Question: if there possibility of collision here? Will we be safe if one of the model names ends with Input?

I'm not sure exactly, I would say no since it has to end with `Select/Include`. What concerns you here?

Reply:
> Also, do Select types pass through Input class in general? From Model.ts it seems like they don't

Through InputField, yes

Reply:
Sorry, I meant to say "what if one of the model names ends with `Select`"